balancer-js/src/modules/pools/pool-types/concerns/stable/join.concern.ts

```diff
@@ -118,15 +213,12 @@ export class StablePoolJoin implements JoinConcern {
     ]);
     const values = amountsIn.filter((amount, i) => tokensIn[i] === AddressZero); // filter native asset (e.g. ETH) amounts
```

Comment:
ComposableStable refactor has a `getEthValue` helper we can use here. Can probably move it into a separate utils file that all pool types can reusue.